app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
```

Comment:
- [ ] Great job working on setting up the devise but when I try to sign up I face these 2 errors because of the validations in your user model so I recommend adding inputs in the form to let the user fill them in while registering a  new user so is not prohibited from completing the signup successfully ✅💪🏻🔥♻


![image](https://github.com/AlaaAlsalem/blogaapp/assets/58553711/a21abbe3-3967-464d-ac4e-27e11321fb87)